src/rdpBuilder.js

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
I know we don't have a code style document yet, but let's stick to the "ovirt-dashboard" code style of naming files `long-file-name.js` instead of `longFileName.js`